src/Sulu/Bundle/SearchBundle/Controller/WebsiteSearchController.php

```diff
@@ -109,6 +117,16 @@ public function queryAction(Request $request)
         );
     }
 
+    private function resolveIndexPlaceholders($value)
+    {
+        $resolver = $this->container->get('twig');
```

Comment:
in my opinion the symfony expression language which we use also in the route-bundle. this expressions will be "compiled" and cached. so they are not as much overhead as twig. see http://symfony.com/doc/current/components/expression_language.html